app/browser/reducers/historyReducer.js

```diff
@@ -126,6 +127,24 @@ const historyReducer = (state, action, immutableAction) => {
         break
       }
 
+    case appConstants.APP_REMOVE_HISTORY_DOMAIN: {
+      const domain = action.get('domain')
+
+      if (!domain) {
+        break
+      }
+
+      historyState.getSites(state).forEach(something => {
```

Comment:
please use a descriptive variable name like `historySite` instead of `something`

Reply:
Oh woops, forgot I left that in there. Nice catch!